src/internal/transactionenv/txncontext/context.go

```diff
@@ -172,7 +178,8 @@ type PpsPropagater interface {
 // commitsets at the end of a transaction.  It is defined here to avoid a
 // circular dependency.
 type PpsJobStopper interface {
-	StopJobs(commitset *pfs.CommitSet)
+	StopJobSet(commitset *pfs.CommitSet)
```

Comment:
We are going to remove the commitset squash / drop anyways, so that will be the case longer term. For 2.5.x, I think it makes sense to leave the commitset squash / drop unchanged.